tests/integration/test_arrowflight_interface/test.py

```diff
@@ -504,9 +504,12 @@ def test_table_function():
 
     node.query("INSERT INTO mytable VALUES (1, 'a'), (2, 'b')")
 
-    # FIXME
-    assert "Failed to get table schema" in node.query_and_get_error(
-        "SELECT * FROM arrowFlight(flight1, dataset = 'mytable')"
-    )
+    assert node.query(
```

Comment:
Maybe we should write tests for the "unhappy" cases too:
1. expired ticket/poll descriptor
2. non existing ticket/poll descriptor
3. malformatted ticket/poll descriptor

I checked the other tests, and I think `test_doput_invalid_query` is not checking the error incase of `SELECT` query in `DoPut`, more likely "Select from a non existing table in `DoPut`". Maybe it worth to check `SELECT` in combination with an existing table in `DoPut`.